src/main.rs

```diff
@@ -37,6 +39,14 @@ struct Args {
     /// If not passed, all Wordle games are run.
     #[clap(short, long)]
     games: Option<usize>,
+
+    /// Launch in Interactive mode.
+    ///
+    /// If not passed, the program will run in automatic mode.
+    ///
+    /// If passed, the --games argument is ignored.
```

Comment:
Ah, you can have `clap` enforce this for you using [argument relations](https://docs.rs/clap/latest/clap/_derive/_tutorial/index.html#argument-relations).

Reply:
Hope what I have done is correct

Reply:
Oh, you can just change the `#[clap(short, long)]` line to:
```rust
#[clap(short, long, conflicts-with = "games")]
```
I think?

Reply:
It works, but is `conflicts_with`.